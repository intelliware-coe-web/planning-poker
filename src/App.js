import React from 'react';
import './App.scss';
import {BrowserRouter as Router, Route} from "react-router-dom";
import Login from "./pages/Login"
import Rooms from "./pages/Rooms";

function App() {
    return (
        <Router>
            <div className="app">

                <div className="component-wrapper">
                    <Route exact path="/" component={Login} />
                    <Route path="/login" component={Login} />
                    <Route path="/rooms" component={Rooms} />
                </div>

            </div>
        </Router>
    );
}

export default App;
