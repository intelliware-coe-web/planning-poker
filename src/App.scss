body {
  text-align: center;
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.component-wrapper {
  margin: 0 15px;
}