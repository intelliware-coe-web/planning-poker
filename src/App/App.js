import { HashRouter as Router, Route } from "react-router-dom";
import React from 'react';
import Host from './../Host/Host'
import Meetings from '../Meetings/Components/Meetings/Meetings';
import Estimate from './../Estimate/Estimate';
import AddMeeting from '../Meetings/Components/AddMeeting/AddMeeting';
import CreateUser from '../User/Components/CreateUser';
import Ticket from '../Ticket/Ticket';
import Story from '../Story/Story';

export default function App() {

  return (
    <Router>
      <Route exact path="/" component={CreateUser}/>
      <Route path="/host" component={Host} />
      <Route path="/meetings/" component={Meetings} />
      <Route path="/meeting/add" component={AddMeeting} />
      <Route path="/estimate/" component={Estimate} />
      <Route path="/ticket/" component={Ticket} />
      <Route path="/story/:storyId" component={Story} />
    </Router>
  );
}
