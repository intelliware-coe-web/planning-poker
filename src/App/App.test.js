import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import {Provider} from 'react-redux';
import configureMockStore from 'redux-mock-store';

const mockStore = configureMockStore();
const store = mockStore(
    {
        user: {
            user: null,
            error: null
        }
    });

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(
      <Provider store={store}>
        <App />
      </Provider>,
      div
  );
  ReactDOM.unmountComponentAtNode(div);
});

it('renders correctly', () => {
  expect(App()).toMatchSnapshot(); 
});
