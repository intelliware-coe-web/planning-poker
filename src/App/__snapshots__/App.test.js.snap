// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Memo(ConnectFunction)
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(CreateUser)",
        "type": [Function],
      }
    }
    exact={true}
    path="/"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(Host)",
        "type": [Function],
      }
    }
    path="/host"
  />
  <Route
    component={[Function]}
    path="/meetings/"
  />
  <Route
    component={[Function]}
    path="/meeting/add"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(Estimate)",
        "type": [Function],
      }
    }
    path="/estimate/"
  />
  <Route
    component={[Function]}
    path="/ticket/"
  />
  <Route
    component={[Function]}
    path="/story/:storyId"
  />
</Memo(ConnectFunction)>
`;
